maintainers/team-list.nix

```
@@ -142,6 +142,7 @@ with lib.maintainers; {
       # gares has no entry in the maintainers list
       siraben
       vbgl
+      Alizter
```

```suggestion
      alizter
```

Also, please move this to a separate commit.